app/stac_api/models.py

```diff
@@ -579,13 +605,13 @@ class Asset(models.Model):
     )
     # using "name" instead of "id", as "id" has a default meaning in django
     name = models.CharField('id', unique=True, max_length=255, validators=[validate_name])
-    file = models.FileField(upload_to=get_upload_to_asset_path, null=True, blank=True)
+    file = models.FileField(upload_to=upload_asset_to_path_hook, max_length=255)
```

Comment:
Would it make sense to limit the allowed file types (or at least to blacklist some file types) to prevent potentially malicious files from being uploaded?

Reply:
The file type would not block malicious, as a malware could be renamed with an allowed file extension. However this could be a good idea to have a validation, that's validate the file extension with the media type and also having the asset name validated with the media type, I've just opened a jira ticket for this: https://jira.swisstopo.ch/browse/BGDIINF_SB-1566